### ISSUE TITLE
HeldItem-Slot not working on world change

### ISSUE THREAD

**Issue by juancruzdal** [RockinChaos/ItemJoin#317]
Plugin version: 5.0.6-RELEASE-b395
Spigot: Paper 1.8.8

I set in the config this:
```
Settings:
  HeldItem-Slot: 2
  HeldItem-Triggers: JOIN, WORLD-SWITCH
  DataTags: true
```

**Comment by RockinChaos** [comment]
That doesn't exist in the release version, it was added in the snapshots.

http://ci.craftationgaming.com/job/ItemJoin/484/

**Comment by juancruzdal** [comment]
The problem still's in that snapshot

**Comment by RockinChaos** [comment]
Sorry about that, resolved in the latest snapshot; http://ci.craftationgaming.com/job/ItemJoin/488/

**Comment by RockinChaos** [comment]
This bug has been officially fixed in the release of ItemJoin v5.0.7.
You can grab it here; Download: https://www.spigotmc.org/resources/itemjoin.12661/download?version=354306

Please read the changelog as the documentation has not been updated yet;
https://github.com/RockinChaos/ItemJoin/wiki/Recent-Changes
